src/wiseagents/wise_agent.py

```diff
@@ -18,6 +18,13 @@ class WiseAgent(yaml.YAMLObject):
     ''' A WiseAgent is an abstract class that represents an agent that can send and receive messages to and from other agents.
     '''
     yaml_tag = u'!wiseagents.WiseAgent'
+
+    def __new__(cls, *args, **kwargs):
+        '''Create a new instance of the class, setting default values for the instance variables.'''
+        obj = super().__new__(cls)
+        obj._collection_name = "wise-agent-collection"
```

Comment:
Should the default values for `llm`, `vector_db`, and `graph_db` be set here too?